### PR TITLE
Fix #1344 - Default value for link block

### LINKED ISSUE #1344: make the link element either have no placeholders, or prefilled defaults
adding a link element, and editing it, suggests it has default values "mozilla.org" and "mozilla". As a demo user, that's good enough for me and I hit save. Instead of doing what I expect (make a link "mozilla" to "mozilla.org"), this now gives me what looks like a broken element.

We should probably turn those placeholders into actual values, so that if a user hits "edit", we copy the values over from `placeholder` to `value`, and now if the user saves, they have a working link element.

(note that we do this for other elements already, like the phone element)

### PR DESCRIPTION
**Pull request by alicoding**
Closes #1344

### CODE REVIEW

**Comment by alicoding** [comment]
@Pomax R?

File: blocks/link/index.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ module.exports = {
             link: {
                 label: 'URL',
                 type: 'input',
-                value: '',
+                value: 'https://mozilla.org',
                 placeholder: 'https://mozilla.org'
```
Comment (Pomax) [inline blocks/link/index.js]: value overwrites placeholder, so these can go with this change.

File: blocks/link/index.js [hunk 2]
```diff
@@ -8,13 +8,13 @@ module.exports = {
...
             },
             text: {
               label: 'Link Text',
               type: 'input',
-              value: '',
+              value: 'Mozilla',
               placeholder: 'Mozilla'
```
Comment (Pomax) [inline blocks/link/index.js]: overwritten by value

**Comment by alicoding** [comment]
r+ by @Pomax on IRC

**Pull request merged**